admin_site/templates/wagtailadmin/login.html

```diff
@@ -63,7 +63,10 @@
       }
 
       hiddenEls.forEach((el) => el.hide());
+      $('.login-form').attr('action', '/auth/complete/password/');
```

Comment:
I trust this is necessary, and that making this change in the actual django template would be awkward?

Reply:
I don't like this either, but this is the best way we came up with Juha. The form action url is set here and I can't think of a sensible way to modify it: https://github.com/wagtail/wagtail/blob/main/wagtail/admin/templates/wagtailadmin/login.html#L28